backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/ObjCExport.kt

```diff
@@ -34,6 +35,12 @@ internal class ObjCExportedInterface(
         val mapper: ObjCExportMapper
 )
 
+data class ObjCExportedStubs(
```

Comment:
This file is a wrong place for this class.
The file generally covers high-level workflow for generating Obj-C export in compiler. And the latter doesn't even use this class.

Reply:
👌